dask_ml/metrics/pairwise.py

```diff
@@ -39,7 +46,13 @@ def pairwise_distances_argmin_min(
     return argmins, mins
 
 
-def pairwise_distances(X, Y, metric="euclidean", n_jobs=None, **kwargs):
+def pairwise_distances(
+    X: ArrayLike,
+    Y: ArrayLike,
+    metric: str = "euclidean",
+    n_jobs: Optional[int] = None,
+    **kwargs
```

Comment:
Does mypy need anything special for `**kwargs`?

Reply:
https://github.com/python/mypy/issues/5876